kitchen-tests/cookbooks/end_to_end/Berksfile

```diff
@@ -1,3 +0,0 @@
-source "https://supermarket.chef.io"
-
-metadata
```

Comment:
why did this disappear?

Reply:
Because it did not seem in my testing to be used at all